doc/source/deploying-on-slurm.rst

```diff
@@ -26,38 +26,45 @@ Clusters managed by Slurm may require that Ray is initialized as a part of the s
   ip_prefix=$(srun --nodes=1 --ntasks=1 -w $node1 hostname --ip-address) # Making address
   suffix=':6379'
   ip_head=$ip_prefix$suffix
+  redis_password=$(python -u -c "import random; print(random.randrange(10**12))")
```

Comment:
nit - use uuid instead?

Reply:
The command `uuidgen` works both on my MacBook Pro as well as on the Berkeley Savio cluster.

Example:
```
$ uuidgen
73B96B27-93A9-4CF2-B137-590EE59DEC66
```